static/app/components/events/interfaces/breadcrumbs/breadcrumbs.tsx

```diff
@@ -150,27 +137,22 @@ function Breadcrumbs({
         </Time>,
         '',
       ]}
-      isEmpty={!breadcrumbs.length}
+      isEmpty={!sortedBreadcrumbs.length}
       {...emptyMessage}
     >
       <Content ref={contentRef}>
         <AutoSizer disableHeight onResize={updateGrid}>
           {({width}) => (
             <StyledList
-              ref={(el: List | null) => {
-                listRef = el;
```

Comment:
On other days I noticed that this was not working anymore. Thanks for fixing it! 🙏